paddle/framework/CMakeLists.txt

```diff
@@ -55,5 +55,6 @@ cc_library(paddle_pybind SHARED
     recurrent_op
     uniform_random_op
     gaussian_random_op
+    lookup_table_op
```

Comment:
The lookup_table_op and other ops defined here in `framework/CMakeLists.txt` are targets defined in `operators/CMakeLists.txt`. This implies a cyclic dependency --
 `framework` and `operators` depend on each other.  Can we remove this cyclic dependency?